src/db/manifest.rs

```diff
@@ -129,49 +134,30 @@ mod tests {
     #[test]
     fn test_manifest_add_log() {
         let mut manifest = Manifest::new();
-        manifest.add_log("test".to_owned());
-        manifest.add_log("test2".to_owned());
-        manifest.add_log("test3".to_owned());
+        manifest.add_log("test".into());
```

Comment:
Can the into not be inferred by the compiler?

Reply:
`to_owned()` and `into()` do the same thing in this case, just with 1/2 the characters. Which makes certain things that are 1 logical idea fit on 1 line with rust fmt.  When working on the unit tests it was just crazy that 3 operations took up the whole screen.

I feel like if we do some generic stuff with add_log then it should be able to take Strings, &strs, and Cows.  I just don't wanna add extra allocation to support unit test ergonomics.